sdks/python/apache_beam/coders/stream.pyx

```diff
@@ -39,7 +39,7 @@ cdef class OutputStream(object):
     if self.data:
       libc.stdlib.free(self.data)
 
-  cpdef write(self, bytes b, bint nested=False):
+  cpdef write(self, b, bint nested=False):
```

Comment:
<!--thread_id:cc_178620576_t; commit:57a73055b9447e7a1fe6ad674673254d9a614eeb; resolved:0-->
<!--section:context-quote-->
> **RobbeSneyders** wrote:
> The bytes type has been removed from this module for now, since a mismatch between the cython bytes type and future bytes type results in a TypeError: expected bytes, got newbytes.
> I have tried replacing bytes with a memory view as explained [here](http://cython.readthedocs.io/en/latest/src/tutorial/strings.html#accepting-strings-from-python-code), but this resulted in a packaging error.
> Any help on this would be appreciated.

<!--section:body-->
CC: @robertwb, who could provide more guidance.